core/templates/domain/utilities/language-util.service.spec.ts

```diff
@@ -184,4 +184,23 @@ describe('Language util service', function() {
       text: 'български'
     }]);
   });
+
+  it('should throw error while autogenerating audio language with invalid code',
+    () => {
+      expect(function() {
+        lus.getAutogeneratedAudioLanguages('hii');
```

Comment:
I think `getAutogeneratedAudioLanguages` returns lanaguge_codes so it should not accept langauge_code as param. Can you please check the function and update the test a(And test message)?